compiler/rustc_interface/src/passes.rs

```diff
@@ -321,6 +321,31 @@ fn early_lint_checks(tcx: TyCtxt<'_>, (): ()) {
     )
 }
 
+fn env_var<'tcx>(tcx: TyCtxt<'tcx>, key: &'tcx OsStr) -> Option<&'tcx OsStr> {
+    let value = std::env::var_os(key);
```

Comment:
```suggestion
    let value = env::var_os(key);
```
Style nit: could you import instead of using 3+ segment paths?